base-image/build.sh

```diff
@@ -7,6 +7,11 @@ BUILD_DIR='/home/builder/tmp'
 mkdir -p $BUILD_DIR
 cp -r $SOURCE_DIR/* $BUILD_DIR/
 cd $BUILD_DIR
-source oe-core/oe-init-build-env build bitbake
-MACHINE=$TARGET_MACHINE bitbake core-image-minimal
-cp --dereference tmp-glibc/deploy/images/$TARGET_MACHINE/core-image-minimal-$TARGET_MACHINE.tar.gz  $DEST_DIR/rootfs.tar.gz
+sed -i "s#.*DL_DIR ?=.*#DL_DIR ?= \"$SHARED_DOWNLOADS\"#g" build/conf/local.conf
+sed -i "s#.*SSTATE_DIR ?=.*#SSTATE_DIR ?= \"$SHARED_SSTATE\"#g" build/conf/local.conf
```

Comment:
Can't we set them in the same manner as `MACHINE`? By prefixing the bitbake command?